packages/twenty-server/src/engine/core-modules/page-layout/entities/page-layout.entity.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  Column,
+  CreateDateColumn,
+  DeleteDateColumn,
+  Entity,
+  Index,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  Relation,
+  UpdateDateColumn,
+} from 'typeorm';
+
+import { PageLayoutType } from 'src/engine/core-modules/page-layout/enums/page-layout-type.enum';
+import { Workspace } from 'src/engine/core-modules/workspace/workspace.entity';
+import { ObjectMetadataEntity } from 'src/engine/metadata-modules/object-metadata/object-metadata.entity';
+
+@Entity({ name: 'pageLayout', schema: 'core' })
+@Index('IDX_PAGE_LAYOUT_WORKSPACE_ID', ['workspaceId'])
+@Index('IDX_PAGE_LAYOUT_OBJECT_METADATA_ID', ['objectMetadataId'])
+export class PageLayoutEntity {
```

Comment:
To ensure no optionals fields are added in the entity we can apply the same pattern than in FieldMetadataEntity
```ts
export class FieldMetadataEntity<
  T extends FieldMetadataType = FieldMetadataType,
> implements Required<FieldMetadataEntity>
```